### PR TITLE
Quarto Python example 

### PR DESCRIPTION
**Pull request by leesahanders**
Adding an example for quarto python using the jupyter engine. Bonus lightbox extension because I love it so much. 

Overall I think the bones are there (lots of lil' gotchas noted on the readme page), but there are probably a ton of improvements so I look forward to feedback. 

Especially: That requirements.txt file is gnarly. I cut it down a little but would love help figuring out the optimal minimum requirements this example actually needs for working.

### CODE REVIEW

**Comment by leesahanders** [comment]
Tested deployment (using rsconnect) just to make sure nothing broke going from my personal repo into this one: <https://colorado.rstudio.com/rsc/connect/#/apps/ba8c450a-0779-4aad-bad5-01875b3d9dfd/access/5785>

**Review by edavidaja — CHANGES_REQUESTED** [review]
Looking good!

A few things:
- could you remove the files in `.ipynb_checkpoints/` from the manifest?
- I think (but am not certain) that the `quarto-python-jupyter_files` and `quarto-python-files` should be created at render time; if so, could you remove them from the manifest?
- could you generate the requirements file yourself? If you did it in a virtual environment the results should be the same, but I don't want to assume.
- if you're going to [hotlink to the image on wikimedia](https://commons.wikimedia.org/wiki/Commons:Reusing_content_outside_Wikimedia/technical#Hotlinking) you may want to do so in their recommended fashion. Alternatively, you may want to make the images part of this folder.

**Comment by leesahanders** [comment]
Deployments working, woohoo!

git backed: https://colorado.rstudio.com/rsc/connect/#/apps/9dbe7eb2-f51b-478e-b776-ce9dfce7daa2/access/5811 
rsconnect: https://colorado.rstudio.com/rsc/connect/#/apps/d786f16e-c19d-4e8e-ae42-8e24e54bae0e/access/5812

File: README.md [hunk 1]
```diff
@@ -59,9 +61,67 @@ For reticulated content, set the `RETICULATE_PYTHON` environment variable to poi
...
+Upgrade pip and then install needed packages: 
+```
+python -m pip install --upgrade pip wheel setuptools
+python -m pip install rsconnect-python
+```
+
+Leave a virtual environment with: 
+```
+deactivate
+```
```
Comment (edavidaja) [inline README.md:127]:
```suggestion

```

I'm not sure this is necessary here.

**Review by edavidaja — APPROVED** [review]
I trimmed some stuff from the top level readme but otherwise I think this is good to go.

**Pull request merged**